core/src/main/scala/kafka/cluster/Partition.scala

```diff
@@ -180,6 +180,15 @@ class Partition(val topicPartition: TopicPartition,
     }
   }
 
+  def oneAboveMinIsr: Boolean = {
```

Comment:
Minor thing, do we want the name to be isOneAboveMinIsr instead since this is a Boolean?